packages/link/src/Component.tsx

```diff
@@ -84,6 +84,7 @@ export const Link = forwardRef<HTMLAnchorElement, LinkProps>(
                 className,
             ),
             'data-test-id': dataTestId,
+            rel: restProps.target === '_blank' ? 'noreferrer noopener' : undefined,
```

Comment:
Надо оставить возможность самому управлять rel
```suggestion
            rel: !restProps.rel && restProps.target === '_blank' ? 'noreferrer noopener' : restProps.rel,
```

Reply:
> Надо оставить возможность самому управлять `rel`

так есть такая возможность) если передали сверху rel, то он просто перезапишет этот rel
вот в этом месте `<a {...componentProps} {...restProps}`

Reply:
можно тест на это написать :)